metagraph/src/annotation/row_diff_builder.cpp

```diff
@@ -315,7 +317,8 @@ void convert_batch_to_row_diff(const std::string &pred_succ_fprefix,
                 = std::max((uint64_t)2,  //CWQ needs a buffer size of at least 2
                            std::min((uint64_t)1'000'000, sources[i].num_relations()));
         targets_size[i].assign(sources[i].num_labels(), 0U);
-        set_rows[i].resize(sources[i].num_labels());
+        set_rows_fwd[i].resize(sources[i].num_labels());
+        set_rows_bwd[i].resize(sources[i].num_labels());
```

Comment:
keep empty lines, otherwise it gets too dense

```suggestion
        set_rows_bwd[i].resize(sources[i].num_labels());

```